File: node/1012.js
//https://www.beecrowd.com.br/judge/pt/problems/view/1011

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [a, b, c] = lines.shift().split(" ");

const PI = 3.14159

console.log(`TRIANGULO: ${(a * c / 2).toFixed(3)}`);
console.log(`CIRCULO: ${ (PI * Math.pow(c, 2)).toFixed(3)}`)
console.log(`TRAPEZIO: ${((parseFloat(a) + parseFloat(b)) * c / 2).toFixed(3)}`)
console.log(`QUADRADO: ${(b * b).toFixed(3)}`)
console.log(`RETANGULO: ${(a * b).toFixed(3)}`)

File: lua/1012.lua
local entry = io.read()
local t = {}
for value in string.gmatch(entry, "[^%s]+") do
    table.insert(t, value)

end
local A = t[1]
local B = t[2]
local C = t[3]

print("TRIANGULO: "..string.format("%.3f", A*C/2))
print("CIRCULO: "..string.format("%.3f", C^2 * 3.14159))
print("TRAPEZIO: "..string.format("%.3f", (A+B)*C/2))
print("QUADRADO: "..string.format("%.3f", B*B))
print("RETANGULO: "..string.format("%.3f", A*B))